app/Http/routes.php

```diff
@@ -321,8 +321,19 @@
     Route::post('api/review/organization' , 'API\OrganizationReviewAPIController@store  ');
 
     Route::resource('api/organization','API\OrganizationAPIController', ['only' => [
-        'index', 'show',
+        'index', 'show', 'update',
     ]]);
+
```

Comment:
All these route should go before the resource. Check the order of the routes above and make them the same